src/project/SidebarView.js

```diff
@@ -54,12 +54,11 @@ define(function (require, exports, module) {
         $projectTitle,
         $projectFilesContainer,
         $workingSetViewsContainer;
-
     var _cmdSplitNone,
         _cmdSplitVertical,
         _cmdSplitHorizontal;
-
-    /**
+    var MinimumSidebarWidth = 200 ; 
```

Comment:
Indicate what the units are, should be `minimumSidebarWidthPX = 200;`.  Also get rid of the extra space before the `;`.  Add a newline (i.e., empty line) after this to separate the variables.